### PR TITLE
style, consts, and bower comps.

### PR DESCRIPTION
**Pull request by stephenplusplus**

### CODE REVIEW

File: todomvc/js/app.js [hunk 1]
```diff
@@ -9,25 +9,6 @@ module TodoApp {
...
-		}
-
-		// Toggle the `completed` state of this todo item.
-		toggle() {
-			this.save({
-				completed: !this.get('completed')
-			});
-		}
-	}
-
```
Comment (stephenplusplus) [inline todomvc/js/app.js:30]: This appeared to be an extra.

File: todomvc/index.html [hunk 2]
```diff
@@ -1,68 +1,65 @@
...
+			</li>
+		</ul>
+		<% if (completed) { %>
+		<button id="clear-completed">Clear completed (<%= completed %>)</button>
+		<% } %>
+	</script>
+	<script src="bower_components/jquery/jquery.js"></script>
+	<script src="bower_components/lodash/lodash.js"></script>
+	<script src="bower_components/backbone/backbone.js"></script>
+	<script src="bower_components/backbone.localStorage/backbone.localStorage.js"></script>
```
Comment (stephenplusplus) [inline todomvc/index.html:55]: Made `backbone.localStorage` a bower component.

File: todomvc/index.html [hunk 3]
```diff
@@ -1,68 +1,65 @@
...
-			traceur.options.experimental = true;
-		</script>
-		<!--
-		<script src="js/es6-module-loader.js"></script>
-		<script type="text/traceur" src="js/todoApp.js"></script>
-	-->
-		<script type="text/traceur" src="js/app.js"></script>
-	</body>
+	<script src="js/lib/traceur/traceur.js"></script>
+	<script src="js/lib/traceur/bootstrap.js"></script>
```
Comment (stephenplusplus) [inline todomvc/index.html:58]: Renamed `js/libs` to `js/lib` :dancer:

File: todomvc/bower_components/todomvc-common/base.css [hunk 4]
```diff
@@ -61,7 +61,7 @@ body {
 	font-style: italic;
 }
 
-#todoapp input:-moz-placeholder {
+#todoapp input::-moz-placeholder {
```
Comment (stephenplusplus) [inline todomvc/bower_components/todomvc-common/base.css:64]: todomvc-common 0.1.8!

**Comment by passy** [comment]
:+1:

**Comment by addyosmani** [comment]
:+1:

**Comment by addyosmani** [comment]
Nice work, @stephenplusplus!

**Pull request merged**